util/stream_dataset.py

```diff
@@ -618,11 +630,17 @@ def _execute_algorithm(self, parameter, kwargs):
         log.debug('<%s> _execute_algorithm Keyword Args %r', self.request_id, sorted(kwargs))
 
         try:
-            if func.function_type == 'PythonFunction':
+            if func.function_type == 'PythonFunction' and func.owner != '__builtin__':
```

Comment:
I'd like to see an example of this in code or release notes. This should be a very useful new feature.

Reply:
will update release notes with example